configd/src/apps/su/main.cpp

```diff
@@ -28,6 +28,10 @@ int main(int argc, char** argv)
     gid_t g = p->pw_gid;
     uid_t u = p->pw_uid;
 
+    gid_t grouplist[256];
+    int group_arr_sz = 256;
+    int ggl = getgrouplist(username, g, grouplist, &group_arr_sz);
```

Comment:
This might trigger a warning when compiling on darwin (gid_t is not int, thus we have a pointer type mismatch).